### DISCUSSION TITLE
can I use operators with config variables?

### DISCUSSION THREAD

**Discussion by Hifilo** [ohmyzsh/ohmyzsh#11641] [Help] [answered]
for example could something like this:

`ZSH_THEME_RANDOM_CANDIDATES=!( "imajes" "gozilla" "smt")` 

be used to load all themes at random except the ones I specify?

**Comment by carlosala** [comment]
That is not possible in `zsh`.

**Comment by mcornella** [comment, marked as answer]
There is no such syntax in zsh but for this specific use case you can use this other variable: [`ZSH_THEME_RANDOM_IGNORED`](https://github.com/ohmyzsh/ohmyzsh/wiki/Settings#zsh_theme_random_ignored).
